src/nfs/nfs_server_impl.cpp

```diff
@@ -208,9 +207,9 @@ void nfs_service_impl::on_get_file_size(
 
             struct stat st;
             if (0 != ::stat(file_path.c_str(), &st)) {
-                LOG_ERROR("{nfs_service} get stat of file %s failed, err = %s",
-                          file_path.c_str(),
-                          strerror(errno));
+                LOG_ERROR_F("[nfs_service] get stat of file {} failed, err = {}",
+                            file_path,
+                            strerror(errno));
```

Comment:
it's recommend to use `dsn::utils::safe_strerror` instead.

Reply:
done